annotation/.env

```diff
@@ -17,9 +17,17 @@ S3_PROVIDER="minio"
 
 KEYCLOAK_HOST="http://dev2.badgerdoc.com/"
 
+ASSETS_SERVICE_SCHEME=http
```

Comment:
We have begun migrating to a single `.env` file for the docker-compose environment. It isn't logical to support both a global `.env` and individual ones for each microservice. If new variables are needed for unit testing, should they be mocked?